### PR TITLE
Saleor 5591 fixing cla for attributes test 3.1

### PR DESCRIPTION
**Pull request by Kjaroszew**
I want to merge this change because I cloned changes form 3.0

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://master.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-5591 Add missing tests for attributes](https://app.clickup.com/t/2549495/SALEOR-5591)

File: cypress/integration/configuration/attributes/createContentAttribute.js [hunk 1]
```diff
@@ -109,7 +125,7 @@ filterTests({ definedTags: ["all"] }, () => {
       });
     });
 
-    it("should create attribute without required value", () => {
+    it("should be able to create attribute without require value TC:SALEOR_0522", () => {
```
Comment (bmigirl) [inline cypress/integration/configuration/attributes/createContentAttribute.js:128]: `requiring?`

File: cypress/integration/configuration/attributes/attributes.js [hunk 2]
```diff
@@ -154,7 +170,7 @@ filterTests({ definedTags: ["all"] }, () => {
         });
     });
 
-    it("should create swatch attribute with image", () => {
+    it("should create swatch attribute with imagev. TC:SALEOR_0532", () => {
```
Comment (karola312) [inline cypress/integration/configuration/attributes/attributes.js:173]: Typo -  it is "imagev", it should be "image"

**Review by karola312 — APPROVED** [review]

File: cypress/integration/configuration/attributes/createContentAttribute.js (hunk 1 above)
Comment (Kjaroszew) [reply cypress/integration/configuration/attributes/createContentAttribute.js:128]: I used title assigned to this test our documents  "Admin should be able to create attribute without require value"

**Review by bmigirl — APPROVED** [review]

**Review by karola312 — APPROVED** [review]

**Pull request merged**